cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceAssemblyOperatorTest.java

```diff
@@ -4,6 +4,7 @@
  */
 package io.strimzi.operator.cluster.operator.assembly;
 
+import com.google.common.collect.Lists;
```

Comment:
I don't think we want to add Guava as dependency. Can you solve this with regular Java lists/arrays? From the code it is not clear why is it really needed. This is also what is breaking the CI builds.